src/panoramic/cli/transform/commands.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+from typing import List, Tuple
+
+import click
+from tqdm import tqdm
+
+from panoramic.cli.connections import Connections
+from panoramic.cli.context import get_company_slug
+from panoramic.cli.local.get import get_transforms
+from panoramic.cli.local.writer import FileWriter
+from panoramic.cli.paths import FileExtension, Paths
+from panoramic.cli.print import echo_error, echo_info
+from panoramic.cli.transform.compiler import TransformCompiler
+from panoramic.cli.transform.executor import TransformExecutor
+from panoramic.cli.transform.pano_transform import CompiledTransform, PanoTransform
+
+
+def create_command():
+    echo_info('Scaffolding a new transform...')
+    name = click.prompt('name')
+
+    connections = Connections.load()
+    connection_names = connections.keys() if connections else []
+    connection_base_text = 'connection'
+
+    if len(connection_names) == 0:
+        connection_prompt_text = connection_base_text
+    elif len(connection_names) > 3:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(list(connection_names)[:3])}}},...)'
+    else:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(connection_names)}}})'
+
+    # Assemble target based on input
+    connection = click.prompt(connection_prompt_text)
+
+    target_view_path = click.prompt(f'target: {connection}.', prompt_suffix="")
+    target = f'{connection}.{target_view_path}'
+
+    transform = PanoTransform(name=name, fields=[], target=target)
+    writer = FileWriter()
+    transform_path = Paths.transforms_dir() / f'{transform.name}{FileExtension.TRANSFORM_YAML.value}'
+
+    if Path.exists(transform_path):
+        echo_error(f'Transform {transform_path} already exists')
+    else:
+        writer.write_transform(transform)
+
+
+def exec_command(
+    compile_only: bool = False,
+    yes: bool = False,
+):
+    company_slug = get_company_slug()
+    compiled_transforms: List[Tuple[CompiledTransform, Path]] = []
+
+    transforms_with_path = get_transforms()
```

Comment:
We should check if this is empty and abort before printing "Compiling transforms..."

Reply:
Good point - on it.

Reply:
Changed below https://github.com/panoramichq/panoramic-cli/pull/37/commits/9da198ecff78dac77036e41cbdf1ff0c0ee3077b#diff-46bafe5b81fc3ea1ee019235a05a1e391144a4575a0901331cb2437369029eadR58-R60